README.md

```diff
@@ -626,6 +630,78 @@ We have a small collection of algorithms, one for every day of the month. Scroll
     - [Understanding Recursion](https://stackoverflow.com/a/717839)
 
 
+### **December 25 - Santa's Self-Driving Sleigh** :christmas_tree:
+  - **Problem**
+    - Help Santa bring gifts to the homes of well-behaved children on Christmas Eve by writing the right program for his new Self-Driving Sleigh.
+    - Assume a 10x10 matrix where both Santa and a Child can be anywhere on the border.
+    - The Location of Santa and Child's Locations (Index) are given as input.
+    - Use Randomize function and distance to the Child's index to reach the location.
+    - `Optional:` Use Recursion to solve the problem.
+    - Merry Christmas!
+
+
+  - **Example**
+    ```
+    Santa's Location: (1,0)
+    Child's Location: (9,8)
+    ```
+    <img src="/src/docs/Dec25-Example.png" width="250" height="250"/>
+
+
+### **December 26 - Vowel Square**
+  - **Problem**
+    - The function VowSq(strArr) takes the parameter as a 2D matrix of some size containing alphabets, find if a 2x2 square matrix composed only of vowels exists in the input matrix.   
+    - If a 2x2 square of vowels is found your input should return the top-left position (row-column) of the square.
+    - If no 2x2 square of vowels exists, then return the string unavailable.  
+    - The input matrix should at least be of size 2x2.
+    - Credits to coderbyte.com
+
+
+  - **Example #1**
+    - For ``strArr = ["abcd", "eikr", "oufj"]``
+    - The Matrix should be:
+    ```
+    a b c d
+    e i k r         
+    o u f j  
+    ```
+    ````
+    1-0
+    ````
+
+
+  - **Example #2**
+    - For ``strArr = ["gh", "bf"]``
+    - The Matrix should be:
+    ```
+    g h b f
+    ```
+    ````
+    Unavailable
+    ````
+
+
+### **December 27 - Correct Path**
+  - **Problem**
+    - The function CorrectPath(str) reads str, which will represent the movements made in a 5x5 grid of cells starting from the top left position.  
+    - The characters of the input string are only: r, l, u, d, ?.  
+    - Each of the characters stand for the direction within the grid, for example:  
+      - r = right
+      - l = left
+      - u = up  
+      - d = down
+    - Find what characters the question marks should be in order for a path to go from the top left of the grid to the bottom right without touching previous cells of the path.
+    - Program should display the full string answer that has a path formed from the top left to the bottom right of 5x5 grid.  
+    - There can be only one correct path to an input string and there is at least one question mark in the input string.
+
+
+  - **Example**
```

Comment:
Explanation:

After processing the non-? instructions we are at

|-|-|-|-|-|
|---|---|---|---|---|
|-|-|-|-|-|
|-|-|-|-|-|
|-|-|-|-|-|
|-|-|X|-|-|

So to reach the bottom right box, we need to go right twice.